yaksh/evaluator_tests/test_bash_evaluation.py

```diff
@@ -103,6 +105,12 @@ def test_infinite_loop(self):
         # Then
         self.assertFalse(result.get("success"))
         self.assert_correct_output(self.timeout_msg, result.get("error"))
+        parent_proc = Process(os.getpid()).children()
+        if parent_proc:
+            self.assertFalse(any(Process(parent_proc[0].pid)\
+                .children(recursive=True)))
+
```

Comment:
Can parent_proc[0].pid OR Process(parent_proc[0].pid).children(recursive=True) be assigned to a variable so that it makes the line of code more readable and saves you a line break